st2actions/st2actions/notifier.py

```diff
@@ -120,12 +122,17 @@ def _post_generic_trigger(self, liveaction):
                    'status': liveaction.status,
                    'start_timestamp': str(liveaction.start_timestamp),
                    'action_name': liveaction.action,
```

Comment:
Looks like there is an inconsistency - above we call it `action_ref` and we call it `action_name` here.

We should also change it to `action_ref` here (yeah, I know that's now your change but since you are already changing the code...).